React/Views/RCTScrollView.m

```diff
@@ -844,6 +851,26 @@ - (id)valueForUndefinedKey:(NSString *)key
   return [_scrollView valueForKey:key];
 }
 
+- (void)setOnRefreshStart:(RCTDirectEventBlock)onRefreshStart
+{
+  _onRefreshStart = [onRefreshStart copy];
```

Comment:
This should probably check if onRefreshStart is nil and remove the refreshControl if it is.

Also, if onRefreshStart is not nil, and _scrollView.refreshControl is already not nil, there's no need to create and assign a new UIRefreshControl.